05/snake_for_if.py

```diff
@@ -0,0 +1,16 @@
+pets_start_second = []
+pets_sorted = []
+pets = ['had', 'andulka', 'pes', 'kočka', 'králík']
+copy_pets = pets
+
+for pet in pets:
+    pet = pet[0] + pet[1:]
+    pets_start_second.append(pet[1:])
+    pets_start_second.sort()
```

Comment:
Zoradiť ich v každom kroku for cyklu nemá zmysel, stačí až po prejdení celého for cyklu keď sa ti zoznam naplní celý.